app/src/org/commcare/android/framework/CommCareActivity.java

```diff
@@ -383,15 +382,6 @@ void attemptDismissDialog() {
     }
 
     /**
-     * Handle an error in task execution.
-     */
-    protected void taskError(Exception e) {
-        //TODO: For forms with good error reporting, integrate that
-        Toast.makeText(this, Localization.get("activity.task.error.generic", new String[]{e.getMessage()}), Toast.LENGTH_LONG).show();
-        Logger.log(AndroidLogger.TYPE_ERROR_WORKFLOW, e.getMessage());
-    }

```

Comment:
Used by logic which is now in `ArchivedFormRemoteRestore`, which is only used by developers to load saved forms onto a phone manually. Taking it out because these errors are also logged elsewhere.